### PR TITLE
PDP-1335 : Create a ReadMe for Legal review

### PR DESCRIPTION
**Pull request by skhamkar-tibco**

### CODE REVIEW

File: charts/dp-core-infrastructure/charts/haproxy-ingress/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (pagarwal-tibco) [inline charts/dp-core-infrastructure/charts/haproxy-ingress/LICENSE:1]: this is copied from public haproxy github repo, right?

Comment (skhamkar-tibco) [reply charts/dp-core-infrastructure/charts/haproxy-ingress/LICENSE:1]: yes

Comment (skhamkar-tibco) [reply charts/dp-core-infrastructure/charts/haproxy-ingress/LICENSE:1]: copied from here -> https://github.com/haproxy-ingress/charts

**Review by pagarwal-tibco — COMMENTED** [review]
I hope you also deleted the 'charts/distributed-lock-operator/config' filder

**Comment by skhamkar-tibco** [comment]
> I hope you also deleted the 'charts/distributed-lock-operator/config' filder

yes I deleted it as it was duplicate config for fluentbit

File: charts/dp-core-infrastructure/Chart.yaml [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# Copyright © 2023. Cloud Software Group, Inc.
+# This file is subject to the license terms contained
+# in the license file that is distributed with this file.
+
 apiVersion: v2
 appVersion: 1.0.253
 dependencies:
```
Comment (sasati-tibco) [inline charts/dp-core-infrastructure/Chart.yaml:7]: We need to change the repository to use tibco/tp-helm-charts private repo, which means the dependencies.repository key will use some alias hence need to do these steps
1.1 helm repo add platform-dev https://<private helm repo dns> --username <github username> --password <PAT token>  (can we use "${{ secrets.GITHUB_TOKEN }}" in action script)
1.2 the sub charts should use repository alias. 
@syan-tibco @pagarwal-tibco would the above approach looks good to you ?

**Review by pagarwal-tibco — APPROVED** [review]

**Pull request merged**